validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -156,17 +160,21 @@ private static void addValidatorCountMetric(
 
   @Override
   protected SafeFuture<?> doStart() {
-    forkProvider.start().reportExceptions();
-    validatorIndexProvider.lookupValidators();
-    eventChannels.subscribe(
-        ValidatorTimingChannel.class,
-        new ValidatorTimingActions(
-            validatorStatusLogger,
-            validatorIndexProvider,
-            blockProductionTimingChannel,
-            attestationTimingChannel));
-    validatorStatusLogger.printInitialValidatorStatuses();
-    return beaconNodeApi.subscribeToEvents();
+    return initializationComplete.thenCompose(
+        (__) -> {
+          System.out.println("initialization complete");
```

Comment:
should probably log rather than println...

Reply:
Done.